knora-ontologies/standoff-onto.ttl

```diff
@@ -97,7 +99,19 @@
                               owl:maxCardinality "1"^^xsd:nonNegativeInteger
```

Comment:
for what it's worth, [mozilla](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a) says about `href`:
> This attribute may be omitted (as of HTML5) to create a placeholder link. A placeholder link resembles a traditional hyperlink, but does not lead anywhere. 

but honestly, I don't understand what a _placeholder link_ is